rita/src/rita_client/traffic_watcher/mod.rs

```diff
@@ -1,3 +1,11 @@
+//! Traffic watcher monitors system traffic by interfacing with KernelInterface to create and check
+//! ebtables counters on each per hop tunnel (the WireGuard tunnel between two devices). These counts
```

Comment:
we don't use ebtables anymore, we use iptables and ipset

Reply:
why switch? specifically the iptables lock is a scaling issue.